### ISSUE TITLE
Virtual scrolling + dynamic row height and scroll not an part of header 

### ISSUE THREAD

**Issue by blinovksiyigor** [komarovalexander/ka-table#145]
Hi @komarovalexander. I'm trying to do both virtual scrolling (i have table with 10k+ rows and dynamic row height) and making scrollbar not part of header. That is what i have right now [sandbox](https://codesandbox.io/s/nice-beaver-tlkyl?file=/src/TableWrapper.jsx). Maybe you can help me. Thanks

**Comment by komarovalexander** [comment]
Hi @blinovksiyigor, Thanks for asking here is the fix: https://codesandbox.io/s/eager-goodall-pvm2w?file=/src/TableWrapper.jsx

a couple of notes:
1) please use 6.8.1 version of ka-table - version in the provided example does not have fix for dataRow ref, it was fixed 2 month ago https://github.com/komarovalexander/ka-table/pull/133
2) call initDynamicRowsOptions outside of datagrid because otherwise you will override renderedRowSizes on each render - or use useState instead

Cheers!

**Comment by blinovksiyigor** [comment]
@komarovalexander thanks for help. One more question : is there the way to make virtualScrolling enabled by default in table wrapper, if i didn't pass it from parent as prop virtualScrolling: {}, for some reason it is not working

**Comment by komarovalexander** [comment]
I suppose this is a solution
```
<Table
      {...props}
      virtualScrolling={{
        **enabled:true**,
        ...props.virtualScrolling,
        itemHeight
      }}
```

**Comment by blinovksiyigor** [comment]
@komarovalexander, about virtualScrolling as 

> this should be a solution
> 
> ```
> <Table
>       {...props}
>       virtualScrolling={{
>         enabled:true, //enable by default
>         ...props.virtualScrolling,
>         itemHeight
>       }}
> ```

For some reason not working. https://codesandbox.io/s/mystifying-thunder-mpjzx?file=/src/TableWrapper.jsx

**Comment by komarovalexander** [comment]
that's a good catch! `kaReducer` doe not process `ActionType.ScrollTable`
```
    case ActionType.ScrollTable:
      if (virtualScrolling) {
          const scrollTop = action.scrollTop;
          return {...props, ...{virtualScrolling: { ...virtualScrolling, scrollTop }}};
        }
      break;
```
Seems like the condition should be removed, I will double-check it and will fix this issue this week
Thanks!

**Comment by komarovalexander** [comment]
@blinovksiyigor fixed in `6.8.2` - thanks for reporting that! 
https://codesandbox.io/s/vigilant-ptolemy-rrt36
enjoy `ka-table`!
